pandas/_libs/tslibs/nattype.pyx

```diff
@@ -82,15 +82,15 @@ cdef _nat_rdivide_op(self, other):
 
 def __nat_unpickle(*args):
     # return constant defined in the module
-    return NaT
+    return c_NaT
 
 # ----------------------------------------------------------------------
 
 
 cdef class _NaT(datetime):
-    cdef readonly:
-        int64_t value
-        object freq
+    # cdef readonly:
```

Comment:
can you remove

Reply:
or do we do this elsewhere to remind of the attributes?

Reply:
Yes, this is the pattern we used for _TSObject